pkg/controller/node/node_controller.go

```diff
@@ -128,6 +143,199 @@ func (ctrl *Controller) Run(workers int, stopCh <-chan struct{}) {
 	<-stopCh
 }
 
+func (crtl *Controller) updateMasterNodes(old, cur interface{}) {
+	oldScheduler := old.(*configv1.Scheduler)
+	curScheduler := cur.(*configv1.Scheduler)
+
+	if oldScheduler.Name != "cluster" && curScheduler.Name != "cluster" {
```

Comment:
I think we want `||` here.

Reply:
Yup. Thanks for catching. I will update it.